app/Http/Controllers/Api/RoleController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Api\RolePermissionRequest;
+use App\Http\Requests\Api\RoleRequest;
+use Spatie\Permission\Models\Role;
+
+class RoleController extends Controller
+{
+    public function index()
```

Comment:
please add return types to all functions

Reply:
Sure.